docs/release_notes/ibexa_dxp_v5.0_deprecations.md

```diff
@@ -0,0 +1,575 @@
+<!-- vale VariablesVersion = NO -->
+
+# Ibexa DXP v5.0 renames, deprecations and removals
+
+This page lists backwards compatibility breaks and deprecations introduced in [[= product_name =]] v5.0.
+
+!!! tip "Upgrade to v5"
+
+    For a guide on moving your project to v5.0,
+    see [Update and migration instructions](update_to_5.0.md).
+
+[[= product_name =]] v5.0 introduces further modifications to significant parts of the code to align with the ones introduced in previous versions.
+
+These changes include dropped packages, changing database table and column names, field identifiers, namespaces, function names, and others.
+
+## Dropped packages
+
+[[= product_name =]] v5.0 no longer includes Personalization.
+It's been replaced with Raptor Customer Data Platform that is included as standard. 
```

Comment:
```suggestion
```

On another note, `ibexa/icons` is a dropped package

Reply:
Already removed locally.